h/schemas/forms/admin/organization.py

```diff
@@ -2,21 +2,38 @@
 
 import colander
 from deform.widget import TextAreaWidget, TextInputWidget
+from xml.etree import ElementTree
 
 import h.i18n
 from h import validators
-from h.models.organization import Organization, ORGANIZATION_NAME_MIN_CHARS, ORGANIZATION_NAME_MAX_CHARS
 from h.schemas.base import CSRFSchema
 
 _ = h.i18n.TranslationString
 
 
+ORGANIZATION_NAME_MIN_CHARS = 1
+ORGANIZATION_NAME_MAX_CHARS = 25
+ORGANIZATION_LOGO_MAX_CHARS = 100000
+
+
+def _strip_xmlns(tag):
+    if '}' in tag:
+        return tag.split('}', 1)[1]
+    return tag
+
+
 def validate_logo(node, value):
+    if not (len(value) <= ORGANIZATION_LOGO_MAX_CHARS):
```

Comment:
I removed the if value is None here as it looks like this is never entered if a logo is left blank or removed on the form.

Reply:
Agree!